package.json

```diff
@@ -158,30 +168,34 @@
         "vscode:prepublish": "npm run compile",
         "compile": "npm install && npm run compile-ts && npm run compile-css && npm run compile-js",
         "compile-ts": "tsc -p ./",
-        "compile-css": "npm run compile-css-1 && npm run compile-css-2 && npm run compile-css-3 && npm run compile-css-4 && npm run compile-css-5",
+        "compile-css": "npm run compile-css-1 && npm run compile-css-2 && npm run compile-css-3 && npm run compile-css-4 && npm run compile-css-5 && npm run compile-css-6",
```

Comment:
There isn't a way simpler way to compile all the .scss files?

Reply:
Yes, I can write a short script to enumerate the files that need compiling. Tracking with https://github.com/neo-project/neo-visual-tracker/issues/25